docs/essentials/caching-dependencies-and-directories.md

```diff
@@ -213,21 +217,30 @@ Examples:
 $ checksum package.json 3dc6f33834092c93d26b71f9a35e4bb3
 ```
 
-### Requirements
+### sftp backend
```

Comment:
These sections are added with the same level of "hierarchy indentation" as commands like cache clear — seems wrong.